src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -472,6 +472,8 @@ public enum SslProtocols
         Tls = 192,
         Tls11 = 768,
         Tls12 = 3072,
+        [Obsolete("This value has been deprecated.  It is no longer supported. http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
I'm not sure we have a strict guideline on it, but for me it's:
- A desire to keep the ref essentially a subset of the implementation; it's ok for the implementation to have additional attributes, for example, but it shouldn't have fewer.
- A desire to get the same obsoletion warnings when this is used in the implementation as when it's used from a consumer